catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -349,18 +363,15 @@ private String getWarningMessage(
         break;
       }
     }
-    return warningMsg;
+    return warningMsg + "\\n";
   }
 
-  private String mapAliasAttributes(
+  private List<String> mapAliasAttributes(
       ServiceReference<QueryResponseTransformer> queryResponseTransformer,
-      List<String> requiredAttrList,
       Map<String, Serializable> arguments) {
-    String idName = Metacard.ID;
-    if (arguments.get("columnAliasMap") != null) {
-      Map<String, Serializable> columnAliasMap =
-          (Map<String, Serializable>) arguments.get("columnAliasMap");
-      idName = columnAliasMap.get(idName) == null ? idName : (String) columnAliasMap.get(idName);
+    List<String> requiredAttrList = new ArrayList<>();
+    if (getAliasMap(arguments) != null) {
+      Map<String, Serializable> columnAliasMap = getAliasMap(arguments);
```

Comment:
✏️  could do this instead:
`Map<String, Serializable> columnAliasMap = Optional.ofNullable(getAliasMap()).orElse(Collections.emptyMap());`